src/containers/App/App.js

```diff
@@ -17,10 +20,13 @@ const App = () => {
   const [newPost, setNewPost] = useState(false)
   const [posts, setPosts] = useState([])
   const [query, setQuery] = useState('')
+  const [userID, setUserID] = useState(2)
+  const { loading, error, data } = useQuery(GET_USER_POSTS(userID));
+  // const [updated, setUpdated] = useState(false)
 
   useEffect(() => {
     let mounted = true;
-    if (mounted) setUsers(users.data.attributes.users)
+    if (mounted) setPosts(data)
```

Comment:
I guess I'm a little confused about the way this is being used. Could you explain how this is being used again?